### ISSUE TITLE
Pulled my code out of this repo

### ISSUE THREAD

**Issue by Em1tt** [Boomy639/BekaHot-bot#2]

**Comment by Boomy639** [comment]
you fucking did what?

**Comment by Em1tt** [comment]
I believe I own full ownership to the code I wrote and I'm not gonna allow anyone to take credit for it. Best thing you can do is either rename the repo or just delete it entirely. twat

**Comment by Boomy639** [comment]
I will make you eat your words just wait until tomorrow I am done with you
treating me like im worth nothing

**Comment by Em1tt** [comment]
I never treated you like nothing. I literally took care of you as if I was your fucking dad. Hell, I even bought you games your parents would refuse to buy you. The reason I pulled my code out of this project is because you are no longer a part of it, even though you made the repo. Instead of fighting and stirring up another conflict I was honest with you and told you I don't want to be friends with you anymore. Now, the fact that you either respect that or not just shows what kind of person you are. But the fact that you just want to stir up more conflicts means you're absolutely childish. Don't expect me to feel guilty, in any way, because I walked out of a very toxic friendship and the fact that you're preparing god knows what attack on me, just assures me, and other people who might stumble upon this public repository, that I did the right decision. Goodbye Boomy
